features/src/main/scala/com/salesforce/op/utils/spark/RichRow.scala

```diff
@@ -116,6 +116,27 @@ object RichRow {
     def getFeatureType[T <: FeatureType](f: TransientFeature)(implicit conv: FeatureTypeSparkConverter[T]): T =
       conv.fromSpark(getAny(f.name))
 
+    /**
+     * Converts row to a [[collection.mutable.Map]]
+     *
+     * @return a [[collection.mutable.Map]] with row contents
+     */
+    def toMutableMap: collection.mutable.Map[String, Any] = {
```

Comment:
https://github.com/apache/spark/blob/master/sql/catalyst/src/main/scala/org/apache/spark/sql/Row.scala#L354

Reply:
Oook, so my function is faster, because `getValuesMap` calls `def getAs[T](fieldName: String): T = getAs[T](fieldIndex(fieldName))` for each value, while my function operates on indices.